draft-ietf-rats-eat.md

```diff
@@ -2049,7 +2049,7 @@ This early allocation will presumably complete correctly
 * Claim Name: SW Measurment Results
 * Claim Description: The results of comparing SW measurements to reference values
 * JWT Claim Name: "swresults"
-* Claim Key: TBD275
+* Claim Key: TBD
```

Comment:
The complement to "swevidence". Why should this Claim ever be produced by an Attester? What is the purpose of allowing this for Evidence?